src/components/App.tsx

```diff
@@ -1155,6 +1155,11 @@ class App extends React.Component<AppProps, AppState> {
         return;
       }
       const data = await parseClipboard(event);
+      if (this.props.onPasteFromClipboard) {
```

Comment:
whats the use case of preventing the paste action ? As you mentioned that you want to customise paste event , but could you explain the use case for it ?

Reply:
Sure,
my current use case is to paste an `id` from my application in excalidraw and transform this text in a "controlled" element (and avoid to have the text in the scene). A little video of this feature:
![aa43e718-4433-426c-820c-3407f44aab2a](https://user-images.githubusercontent.com/1761469/114144543-baaa0680-9915-11eb-90f5-69433ace2a34.gif)

We can also used this little hook to implement some custom svg pasting or thing like this.

Reply:
yeah this sounds good 👍